cmake/environment/GithubSshTest.cmake

```diff
@@ -0,0 +1,90 @@
+##############################################################################
+#
+# medInria
```

Comment:
As you understood, this was imported from medInria. Will remove. But it's not important for now.